### PR TITLE
Translate docs/faq/_index.md into Korean

### LINKED ISSUE #834: Translate docs/faq/_index.md into Korean
Translate docs/faq/_index.md into Korean.

Page to translate: https://helm.sh/docs/faq

### PR DESCRIPTION
**Pull request by markruler**
Closes #834 

- https://helm.sh/docs/faq/

#840 추가될 부분도 같이 번역했습니다.

### CODE REVIEW

**Comment by markruler** [comment]
@bluefriday @jmnote @singlemocha

File: content/ko/docs/faq/_index.md [hunk 1]
```diff
@@ -5,121 +5,122 @@ weight: 8
...
+[kubeconfig 파일](https://kubernetes.io/ko/docs/concepts/configuration/organize-cluster-access-kubeconfig/)을
+사용하여 평가됩니다.
+클러스터 관리자는 세세하게 사용자 권한을 제한할 수 있습니다.
+릴리스는 여전히 클러스터 내에 기록되며,
+헬름의 나머지 기능은 그대로 유지됩니다.
+
+### 업그레이드 전략 개선: 3 방향 전략적 병합 패치
+
+헬름 2는 양방향 전략적 병합 패치를 사용했었습니다.
+최근 차트의 매니페스트와 새로 제시된 차트의 매니페스트(`helm upgrade` 중에 제공된 것 중
```
Comment (singlemocha) [inline content/ko/docs/faq/_index.md:56]: 앞부분에 'During an upgrate'가 생략되어 있습니다.
업그레이드 할 때 최근 차트의 매니페스트와 새로 제시된 차트의 매니페스트('helm upgrade' 시에 제공된 매니페스트)를 비교했습니다.

File: content/ko/docs/faq/_index.md [hunk 2]
```diff
@@ -5,121 +5,122 @@ weight: 8
...
+상태를 유지하거나 헬름 릴리스 정보의 중심 허브 역할을 하기 위해
+클러스터 내 운영자에게 의존할 필요가 없다는 것을 알게 되었습니다.
+그 대신 쿠버네티스 API 서버에서 정보를 간단히 가져오고,
+차트 클라이언트 측을 렌더링하며 쿠버네티스에 설치 기록을 저장할 수 있습니다.
+
+틸러의 주요 목표는 틸러 없이도 달성될 수 있습니다.
+그래서 헬름 3와 관련하여 우리가 내린 첫 번째 결정 중 하나는 틸러를 완전히 제거하는 것입니다.
+
+틸러가 사라지면 헬름의 보안 모델이 획기적으로 간소화됩니다.
+헬름 3는 이제 현대 쿠버네티스의 모든 현대적인 보안, 신원 및 인가 기능을 지원합니다. 헬름의 권한은
```
[the remark this answers is not in the document]
Comment (markruler) [reply content/ko/docs/faq/_index.md:46]: 리뷰 감사드립니다. 😊
혹시 '신원 확인 및 인가' 기능은 어떨까요? 쿠버네티스 문서에서도 authorization은 인가라고 번역해서요.

```suggestion
헬름 3는 이제 현대 쿠버네티스의 모든 현대적인 보안, 신원 확인 및 인가 기능을 지원합니다. 헬름의 권한은 
```

File: content/ko/docs/faq/_index.md (hunk 1 above)
Comment (markruler) [reply content/ko/docs/faq/_index.md:56]: 의견 반영해서 수정하겠습니다.

File: content/ko/docs/faq/_index.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (markruler) [reply content/ko/docs/faq/_index.md:46]: 신원 확인으로 수정하겠습니다!

File: content/ko/docs/faq/_index.md [hunk 3]
```diff
@@ -464,35 +480,65 @@ The following table gives the default folder for each of these, by OS:
...
-can access them.
+GKE에 클러스터를 생성하면 SSL 인증서 및 인증 기관을 포함한
+인증서가 제공됩니다. 이러한 파일은 `kubectl`과 `helm`이
+접근할 수 있도록 쿠버네티스 설정 파일에 저장되어야
+합니다(기본값: `~/.kube/config`).
 
-### 헬름 2에서 전환 후, `helm list`에는 릴리스들이 일부만 보여요(또는 안 보여요).
+### 헬름 2에서 마이그레이션한 후, `helm list`에는 릴리스들이 일부만 보여요(또는 안 보여요).
 
 헬름 3는 이제 클러스터 네임스페이스를 사용하여 릴리스들을 구획한다는 사실을 깜빡했을지 모르겠다.
```
Comment (jmnote) [inline content/ko/docs/faq/_index.md:505]:
```suggestion
헬름 3는 이제 클러스터 네임스페이스를 사용하여 릴리스들을 구획한다는 사실을
깜빡하신 것 같습니다.
```

File: content/ko/docs/faq/_index.md [hunk 4]
```diff
@@ -464,35 +480,65 @@ The following table gives the default folder for each of these, by OS:
...
-따라서, 릴리스를 참조하는 모든 명령어에 대해:
+따라서, 릴리스를 참조하는 모든 명령어에 대해 다음 중 하나를 수행해야 합니다:
```
Comment (jmnote) [inline content/ko/docs/faq/_index.md:506]:
```suggestion
릴리스를 참조하는 모든 명령어는 다음 중 하나로 수행해야 합니다:
```

File: content/ko/docs/faq/_index.md [hunk 5]
```diff
@@ -464,35 +480,65 @@ The following table gives the default folder for each of these, by OS:
...
+* `--namespace`/`-n` 플래그를 사용하여 올바른 네임스페이스를 지정합니다.
+* `helm list` 명령어에 대해서는 `--all-namespaces`/`-A` 플래그를 지정합니다.
+
+이는 릴리스를 참조하는 `helm list`, `helm uninstall` 및 기타 모든 `helm` 명령어에 적용됩니다.
+
+### 맥OS에서는 `/etc/.mdns_debug` 파일에 접근합니다. 왜 그런가요?
+
+우리는 맥OS에서 헬름이 `/etc/.mdns_debug`라는 파일에 접근하려고
+하는 경우를 알고 있습니다. 파일이 있는 경우, 헬름은 파일이 실행되는 동안
+파일 핸들을 열어 둡니다.
```
Comment (jmnote) [inline content/ko/docs/faq/_index.md:518]:
```suggestion
맥OS에서는 헬름이 `/etc/.mdns_debug`라는 파일에 접근하려고
하는 것으로 알려져 있습니다. 파일이 있는 경우, 헬름은 실행되는 동안
파일 핸들을 열어 둡니다.
```

**Review by jmnote — COMMENTED** [review]
수고하셨습니다. 일부 수정해보았습니다.

**Comment by markruler** [comment]
@jmnote 꼼꼼한 리뷰 감사드립니다! 의견 반영해서 수정해두었습니다.

**Comment by jmnote** [comment]
lgtm

File: content/ko/docs/faq/_index.md [hunk 6]
```diff
@@ -168,70 +169,83 @@ containers:
...
 
-### 수용능력
+### Capabilities
 
-스테이지 렌더링시에 사용가능한 `.Capabilities` 빌트인 객체가 간소화되었다.
+렌더링 단계에서 사용 가능한 `.Capabilities` 빌트인 객체가
+간소화되었습니다.
 
-[빌트인 객체](/docs/chart_template_guide/builtin_objects/)
+[빌트인 객체](/ko/docs/chart_template_guide/builtin_objects/)
```
Comment (bridgetkromhout) [inline content/ko/docs/faq/_index.md:222]: You do not need to specify this - if it exists, and the view is set to `ko`, it is loaded automatically.

Comment (markruler) [reply content/ko/docs/faq/_index.md:222]: Should I remove all the other "/ko" strings? (L47, L238)

Comment (bridgetkromhout) [reply content/ko/docs/faq/_index.md:222]: In general you can scroll down to the Checks in the pull request and look at the helm-merge preview: https://deploy-preview-841--helm-merge.netlify.app/ in this case. You can choose Korean from the menu. And then you can test and see if it displays as desired without hardcoding `ko` into your URLs.

Comment (markruler) [reply content/ko/docs/faq/_index.md]: > In general you can scroll down to the Checks in the pull request and look at the helm-merge preview: https://deploy-preview-841--helm-merge.netlify.app/ in this case. You can choose Korean from the menu. And then you can test and see if it displays as desired without hardcoding `ko` into your URLs.

It does not load automatically. [link](https://deploy-preview-841--helm-merge.netlify.app/ko/docs/faq/#capabilities)

Comment (bridgetkromhout) [reply content/ko/docs/faq/_index.md]: Okay; I'll merge and check.

File: content/ko/docs/faq/_index.md [hunk 7]
```diff
@@ -1,498 +1,544 @@
...
+헬름 3에서는 특정 릴리스에 대한 정보가 릴리스 자체와 동일한 네임스페이스에
+저장됩니다. 즉, 이제 사용자는 별도의 두 네임스페이스에서
+`helm install wordpress stable/wordpress`를 수행할 수 있으며,
+현재 네임스페이스 컨텍스트를 변경하여 `helm list` 를 조회할 수 있습니다.
+(예: `helm list --namespace foo`)
+
+이렇게 네이티브 클러스터 네임스페이스가 보다 개선되면서
+`helm list` 명령은 기본적으로 모든 릴리스를 나열하지 않습니다.
+대신 현재 쿠버네티스 컨텍스트의 네임스페이스에 있는 릴리스만
+나열됩니다(즉, `kubectel config view --minify`를 실행할 때 표시되는 네임스페이스).
```
Comment (bridgetkromhout) [inline content/ko/docs/faq/_index.md:188]: `kubectel` is a typo - please change it to `kubectl`. Thanks.

File: content/ko/docs/faq/_index.md [hunk 8]
```diff
@@ -1,498 +1,544 @@
...
+이러한 명령어는 예전 동사도 별칭으로 유지하므로
+두 가지 형식에서 계속 사용할 수 있습니다.
+
+### 네임스페이스 자동 생성
+
+존재하지 않는 네임스페이스에 릴리스를 생성할 때 헬름 2가 네임스페이스를 만들었습니다.
+헬름 3는 다른 쿠버네티스 도구의 동작을 따르며 네임스페이스가 없는 경우
+오류를 반환합니다. `--create-namespace` 플래그를 명시적으로 지정하면 헬름 3가
+네임스페이스를 만듭니다.
+
```
Comment (bridgetkromhout) [inline content/ko/docs/faq/_index.md:430]: We added an item that should go here: https://helm.sh/docs/faq/#what-happened-to-chartapiversion

File: content/ko/docs/faq/_index.md [hunk 9]
```diff
@@ -1,498 +1,544 @@
...
+`helm repo add <reponame> <url>` 명령어를 실행해도 출력되는 내용은 없습니다.
+`--no-update` 플래그를 사용하면 저장소가 이미 등록되어 있는 경우 오류를 발생시킵니다.
+
+헬름 3.3.2 이상에서 기존 저장소를 추가하려고 하면 다음 오류가 발생합니다.
+
+이제 기본 동작은 반대가 됩니다. `--no-update`는 이제 무시되며,
+기존 저장소를 교체(덮어쓰기)하려면 `--force-update`를 사용할 수 있습니다.
+
+이는 [헬름 3.3.2 릴리스 노트](https://github.com/helm/helm/releases/tag/v3.3.2)에서
+설명된대로 보안 픽스에 따라 단절적 변경(breaking change)이 있었기 때문입니다.
```
Comment (bridgetkromhout) [inline content/ko/docs/faq/_index.md:544]: We added one more section here: https://helm.sh/docs/faq/#enabling-kubernetes-client-logging

File: content/ko/docs/faq/_index.md (hunk 6 above)
Comment (bridgetkromhout) [reply content/ko/docs/faq/_index.md]: (After the other corrections are made. Thanks!)

**Comment by mattfarina** [comment]
@markruler Can you please resolve the merge conflict

**Review by bridgetkromhout — APPROVED** [review]
Thank you!

**Pull request merged**